packages/markdown/src/utils.ts

```diff
@@ -37,6 +37,8 @@ function renderMarkdown(
     KEEP_CONTENT: false,
     ADD_ATTR: ['target'],
     FORBID_ATTR: ['style', 'class'],
+    FORBID_TAGS: ['style'],
+    ALLOWED_TAGS: ['a', '#text', 'p', 'li', 'ol', 'ul', 'b', 'strong', 'i', 'em'],
```

Comment:
```suggestion
    ALLOWED_TAGS: ['a', '#text', 'p', 'li', 'ol', 'ul', 'b', 'strong', 'i', 'em', 'del', 'code'],
```
Think we'll want to include `del` and `code` as well